NuGet.Config

```diff
@@ -27,6 +27,7 @@
     <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
     <add key="myget.org devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
     <add key="myget.org roslyn-tools" value = "https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+    <add key="dotnet.myget.org msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
```

Comment:
The sum total of our comments on this now exceed the text needed to make it correct.  

---
In reply to: [75894213](https://github.com/dotnet/roslyn/pull/13288#discussion_r75894213) [](ancestors = 75894213,75893767)